### PR TITLE
feat(settings): keyboard shortcut to open app

### PR DESCRIPTION
**Pull request by setchy**
revive #1032

closes https://github.com/gitify-app/gitify/issues/995

Changed the shortcut key to be `CommandOrControl+Shift+G` since `CmdOrCtrl+Alt+G` is taken by Google Drive on macOS (this explains the previous flickering I reported on #1032).

Screenshot 2024-06-17 at 6 07 12 PM

### CODE REVIEW

**Comment by afonsojramos** [comment]
What if we give the ability to set the shortcut they want?

**Comment by setchy** [comment]
> What if we give the ability to set the shortcut they want?

Could that be a separate enhancement?

File: main.js [hunk 1]
```diff
@@ -122,6 +128,27 @@ app.whenReady().then(async () => {
     }
   });
 
+  ipc.on(
+    'gitify:update-keyboard-shortcut',
+    (_, { enabled, keyboardShortcut }) => {
+      console.log('enabled', enabled);
+
+      console.log('keyboardShortcut', keyboardShortcut);
```
Comment (bmulholland) [inline main.js:136]: Should these still be here? :)

**Review by bmulholland — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (setchy) [reply main.js:136]: not at all 🙈

Comment (setchy) [reply main.js]: Removed 🧼 

Great catch @bmulholland  🙇

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**